website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -102,12 +102,15 @@ Avoid using critical Buttons in Toasts. We handle the prominence and importance
 
 ### Links
 
-Use [Standalone Links](/components/link/standalone) when an action takes the user to a new destination (URL). Follow the Standalone Link [usage guidelines](/components/link/standalone#usage) to determine what variant to use.
+Use [Standalone Links](/components/link/standalone) when an action takes the user to a new destination (URL). Follow the Standalone Link [usage guidelines](https://helios.hashicorp.design/components/link/standalone) to determine what variant to use.
```

Comment:
> Follow the standalone link [usage guidelines](https://helios.hashicorp.design/components/link/standalone) to determine what variant “type” to use.

Is this part still necessary since we're now providing them with an official recommendation?